README.md

```diff
@@ -1,15 +1,15 @@
 tmc_gazebo_plugins
 ===========================
 
-概要
+Overview
 ===========================
-[Gazebo](http://gazebosim.org/)用のロボットに関わらず使えるプラグインを管理する。
+Manages the plugin that can be used regardless of the [Gazebo](http://gazebosim.org/) robot.
```

Comment:
The original sentence means
 * This repository manages the plugin for Gazebo
 * The plugin is not depend on robots (I mean, it's "general" plugin. Not only for some specific robots.)
Could you describe these info in here?

Reply:
How about this:

- This repository manages the plugin that can be used with any [Gazebo](http://gazebosim.org/) robot.

Reply:
perfect!

Reply:
OK. :)
I will make the edit and repush.